src/components/organisms/nav_fixed_top/nav_fixed_top.html

```diff
@@ -1,6 +1,6 @@
 <nav class="navbar navbar-fixed-top" role="navigation">
   <div class="navbar-header">
-    <a href="/">
+    <a href="index.html">
```

Comment:
If you use `.`, the URL will just be relative home, not sure if it works on Pantheon though, because reasons.